src/Pages/form/Components/StrokePredictionForm.tsx

```diff
@@ -42,33 +70,98 @@ const StrokePredictionForm: React.FC = () => {
       >
         {({ values, setFieldValue, isSubmitting, errors }) => (
           <Form>
-            <SectionTitle title="Patient Information" />
-            <PatientInformationSection />
-            <SectionTitle title="Chief Complaint" />
-            <ChiefComplaintSection
-              value={values.ChiefComplaint}
-              name="ChiefComplaint"
-              onChange={setFieldValue}
-            />
-            <SectionTitle title="Underlying Disease" />
-            <UnderLyingDiseaseSection />
-            <SectionTitle title="Vital Signs" />
-            <VitalSignsSection />
-            <SectionTitle title="EKG 12 Leads" />
-            <EKG12LeadsSection
-              value={values.EKG12Leads}
-              name="EKG12Leads"
-              onChange={setFieldValue}
-            />
-            <SectionTitle title="NIHSS" />
-            <NIHSSSection
-              values={values.NIHSS}
-              fieldName="NIHSS"
-              onChange={setFieldValue}
-            />
-            <Button disabled={isSubmitting} type="submit">
-              Submit{""}
-            </Button>
+            <ScrollingProvider>
+              <Grid container>
```

Comment:
In my opinion, this grid does not work either. So, please try to removing and clarifying to me.